docker/1.11.0/Dockerfile.gpu

```diff
@@ -74,9 +74,13 @@ WORKDIR /
 COPY $framework_installable .
 COPY $framework_support_installable .
 
-RUN pip install --no-cache-dir -U $framework_installable \
-    $framework_support_installable\
+RUN pip install --no-cache-dir -U \
+    keras==2.2.4 \
+    $framework_support_installable \
     "sagemaker-tensorflow>=1.11,<1.12" \
+    # Let's install TensorFlow separatly in the end to avoid
+    # the library version to be overwritten
+    && pip install --no-cache-dir -U $framework_installable \
```

Comment:
maybe you need  --force-reinstall  here in case tf installed with keras has same version number?